public/index.html

```diff
@@ -80,16 +80,18 @@
             startTimer("pageload-nav");
         </script>
 
-        <div class="wrapper">
-            <div class="outer-circle" id="nav1">
-                <div class="inner-circle"></div>
-            </div>
-            <div class="outer-circle" id="nav2">
-                <div class="inner-circle"></div>
-            </div>
-            <div class="outer-circle" id="nav3">
-                <div class="inner-circle"></div>
-            </div>
+        <div class="wrapper" id="nav-bar">
+            <ul>
+                <li class="outer-circle">
+                    <a href="#one" class="inner-circle" id="nav1"></a>
+                </li>
+                <li class="outer-circle">
+                    <a href="#two" class="inner-circle" id="nav2"></a>
+                </li>
+                <li class="outer-circle">
+                    <a href="#three" class="inner-circle" id="nav3"></a>
+                </li>
+            </ul>
```

Comment:
If you can, please use class instead of id (I don't see style about nav, sorry if I'm wrong)